tests/functional/strategy/test_attacks.py

```diff
@@ -0,0 +1,55 @@
+from brownie import Wei, reverts
+import brownie
+
+
+def test_sandwich_attack(TestStrategy, web3, token, gov, vault, strategist, rando):
+
+    honest_lp = gov
+    attacker = rando
+    balance = token.balanceOf(honest_lp) / 2
+
+    # seed attacker their funds. Same amount as deposited in vault
+    token.transfer(attacker, balance, {"from": honest_lp})
+    bv = print(token.balanceOf(vault) / 1e18)
+    vault.withdraw(vault.balanceOf(honest_lp) / 2, {"from": honest_lp})
```

Comment:
Why is this withdraw happening?

Reply:
This is so the attacker can attack with same balance as in the vault. and honest_lp needs some money to give to strategy for it to return as profit

Reply:
Weird. Why not account for this externally e.g. in the prior transfer to the attacker?

Reply:
When vault is deployed in conftest it is seeded with half of the supply of token. Attacker can't equal amount in the vault and also has profit added. Could do it another way if you'd rather. But I thought attacker doubling vault assets was neat. Now we've changed what we're looking for at the end I don't think it matters. As attacker will get same return % on his funds regardless of what he attacks with.

Reply:
Ok redone the maths and changed the way we do it. see latest commit

Reply:
Ahhhh, this makes sense. Also, I think there's a minting function in the test token? If not, you can add it.

Reply:
Added public mint function to test Token in d98af08